x-pack/plugins/cases/public/components/status/status_popover_button.test.tsx

```diff
@@ -0,0 +1,99 @@
+/*
```

Comment:
This file is the same as `x-pack/plugins/cases/public/components/status/status.test.tsx`